test/job.js

```diff
@@ -46,13 +48,82 @@ describe('/POST Post a new job', () => {
           response.body.should.have
             .property('message')
             .eql('Create the job successful!');
-          response.body.job.should.have.property('id');
-          response.body.job.should.have
-            .property('title')
-            .eql(jobParameters.title);
-          response.body.job.should.have
-            .property('tags')
-            .eql(jobParameters.tags);
+          id = response.body.job.id;
+          done();
+        });
+    });
+  });
+
+  describe('/PUT Update the job', () => {
+    it('Update the job with properly parameters', (done) => {
+      const jobParameters = {
+        id,
+        title: 'ChangeTitleToThisLine',
+        salaryRange: '$450-$700',
+        description: 'Work with customers and dev team',
+        tags: ['English', 'Communicate', 'Technical knowledge'],
+        company: 'Faba',
+        logoURL:
+          'https://images.unsplash.com/photo-1593642632559-0c6d3fc62b89?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=1950&q=80'
+      };
+      chai
+        .request(app())
+        .put('/api/jobs')
+        .set('Authorization', 'Bearer ' + accessToken)
+        .send(jobParameters)
+        .end((err, response) => {
+          response.should.have.status(200);
+          response.body.should.have
+            .property('message')
+            .eql('Update the job successful!');
+          done();
+        });
+    });
+    it('Update the job with non-UUID ID', (done) => {
+      const jobParameters = {
+        id: 'example',
+        title: 'ChangeTitleToThisLine',
+        salaryRange: '$450-$700',
+        description: 'Work with customers and dev team',
+        tags: ['English', 'Communicate', 'Technical knowledge'],
+        company: 'Faba',
+        logoURL:
+          'https://images.unsplash.com/photo-1593642632559-0c6d3fc62b89?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=1950&q=80'
+      };
+      chai
+        .request(app())
+        .put('/api/jobs')
+        .set('Authorization', 'Bearer ' + accessToken)
+        .send(jobParameters)
+        .end((err, response) => {
+          response.should.have.status(400);
+          response.body.should.have
+            .property('message')
+            .eql('The ID is not match format!');
```

Comment:
It's weird for the API to check the ID format. If we cannot find the job to update by the ID value, just return the update is failed. I think this test case and the one below "Update the job with non-exist ID" can be combined into one.